package.json

```diff
@@ -45,7 +45,7 @@
     "@lundalogik/cz-conventional-changelog": "^3.1.0",
     "@lundalogik/lime-icons8": "^2.7.0",
     "@popperjs/core": "^2.9.3",
-    "@rjsf/core": "^4.0.0",
+    "@rjsf/core": "^2.5.1",
```

Comment:
Have we fixed debendabot so it will not just update this again? If not, will exact version here prevent dependabot from updating it? :thinking:

Reply:
I don't think exact version would prevent that, no. I think that's something we can configure in `dependabot.yml` though. I'll have a look 👍

Reply:
@adrianschmidt I was thinking of using the ignore command in the PR that is found in the collapsible section.

https://github.com/Lundalogik/lime-elements/pull/1631

> You can trigger Dependabot actions by commenting on this PR:
> 
> ...
> @dependabot ignore this major version will close this PR and stop Dependabot creating any more for this major version (unless you reopen the PR or upgrade to it yourself)
> @dependabot ignore this minor version will close this PR and stop Dependabot creating any more for this minor version (unless you reopen the PR or upgrade to it yourself)
> @dependabot ignore this dependency will close this PR and stop Dependabot creating any more for this dependency (unless you reopen the PR or upgrade to it yourself)

Reply:
Yeah, I know that's available, but that doesn't feel like a great option. I don't think there's anything telling another dev that comes to the project that you've done that. Dependabot will just ignore a specific dependency for no apparent reason, unless you happen to find the comment where someone gave it the command. I'm adding the ignore to the `dependabot.yml` instead.

Reply:
I agree! I will definitely forget about this in a cycle or two.

From the docs: 
> If you run any of the commands for ignoring dependencies or versions, Dependabot stores the preferences for the repository centrally. While this is a quick solution, for repositories with more than one contributor it is better to explicitly define the dependencies and versions to ignore in the configuration file. This makes it easy for all contributors to see why a particular dependency isn't being updated automatically. 

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/managing-pull-requests-for-dependency-updates

Reply:
I envisioned having to jump through all kinds of hoops, with downgrading, then upgrading again to enable dependabot for that dependency, then downgrading _again_ because we didn't want the upgraded version. But as soon as we changed the version, dependabot triggered for it again, so it's all good now 👌

Very nice to know for the future too, so I won't have to worry next time! 😊